### PR TITLE
add docs for accessing the infrastructure

### PR DESCRIPTION
**Pull request by cybojenix**

### CODE REVIEW

**Comment by cybojenix** [comment]
@georgeschena, rationale for closing?

File: source/documentation/infrastructure/getting-access.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+### Accessing the Bastion
+
+#### Secrets
+
+[Extract the SSH Key secrets][getting-a-secret] for the bastion server:
+
+- staging secret name: `keys/govwifi-staging-bastion-key`
+- production secret name: `keys/govwifi-bastion-key`
+
+Find out the Elastic IPs for the bastion servers. You can do this by going into the AWS console,
```
Comment (phantas) [inline source/documentation/infrastructure/getting-access.md]: Can you add somewhere the AWS account number(s) or link to where these can be found, so one can configure the assume role on AWS console or `aws vault` (but you don't need to explain how to configure this!)?

File: source/documentation/infrastructure/getting-access.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+and find the instances with the Bastion name in.
+
+Remember that there are 2 regions, so there may be more than 2 bastions.
+
+#### SSH Config
+
+It is recommended to set up an ssh config for ease of use. All further instructions will
+assume you use similar naming.
+
+**Note**: The IP addresses have been redacted. Please substitute in the correct IP addresses.
```
Comment (phantas) [inline source/documentation/infrastructure/getting-access.md]: Do you already have an example of an ssh-config in the repo somewhere? If so, you can link to it instead

File: source/documentation/infrastructure/getting-access.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+ssh govwifi-bastion-london-staging
+```
+
+## Databases
+
+There are 4 databases, all currently located in London. 2 for staging, 2 for production.
+
+To access each one, you will need to use their respective credentials and bastion server.
+
+### Admin database
```
Comment (phantas) [inline source/documentation/infrastructure/getting-access.md]: Can you add a one-liner to describe what this database is used for, or what information you can find there?

File: source/documentation/infrastructure/getting-access.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+- Staging: `grep 'admin-db-username' '.private/non-encrypted/secrets-to-copy/govwifi/staging-london/variables.auto.tfvars'`
+
+**Password**: Get the password from the [encrypted terraform secrets][getting-a-secret]:
+
+- Production secret name: `PASSWORD_STORE_DIR=.private/passwords pass show govwifi/wifi-london/secrets.tf | grep admin-db-password`
+- Staging: `PASSWORD_STORE_DIR=.private/passwords pass show govwifi/staging-london/secrets.tf | grep admin-db-password`
+
+Use your favourite GUI, or set up an SSH tunnel.
+
+### Wifi database
```
Comment (phantas) [inline source/documentation/infrastructure/getting-access.md]: Same as above

File: source/documentation/infrastructure/getting-access.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+You should now be able to connect to each of the hosts using ssh.
+
+```sh
+ssh govwifi-bastion-london-staging
+```
+
+## Databases
+
+There are 4 databases, all currently located in London. 2 for staging, 2 for production.
```
Comment (phantas) [inline source/documentation/infrastructure/getting-access.md]: Don't we actually have *8* databases, 4 for staging, 4 for production? Looking at the AWS console, the list below does not include Sign-in database (aka `wifi-[production|staging]-user-[db|rr]`

Comment (cybojenix) [reply source/documentation/infrastructure/getting-access.md]: 10 databases.

**Production**:
- London: 5
  - User (leader + replica)
  - Session (leader + replica)
  - Admin (leader)
- Ireland 1
  - User (replica)

**Staging**:
- London: 3
  - User (leader)
  - Session (leader)
  - Admin (leader)
- Ireland: 1
  - User (replica)

File: source/documentation/infrastructure/getting-access.md (hunk 2 above)
Comment (cybojenix) [reply source/documentation/infrastructure/getting-access.md]: nope, this is the only example

**Pull request merged**